apps/backend/src/data/migrations/1654702006571-create-menu-table.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey
+} from 'typeorm';
+
+export class CreateMenuTable1654702006571 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'menu',
+        columns: [
+          {
+            name: 'id',
+            type: 'int',
+            isGenerated: true,
+            generationStrategy: 'increment',
+            isPrimary: true
+          },
+          {
+            name: 'store_id',
+            type: 'int'
+          },
+          {
+            name: 'name',
+            type: 'varchar'
```

Comment:
please specify length for each varchar/text type
for each object/column you can specify `length` property with the varchar/text maximum length